server/text_generation_server/models/globals.py

```diff
@@ -6,8 +6,7 @@
 from text_generation_server.utils.log import log_master
 
 PREFIX_CACHING = os.getenv("USE_PREFIX_CACHING", False)
```

Comment:
nit to potentially avoid env var issues
```suggestion
PREFIX_CACHING = os.getenv("USE_PREFIX_CACHING", "").lower() == "true"
```